proj/src/main/webapp/script.js

```diff
@@ -406,18 +411,27 @@ function makeChatPreview(name, messageObj, tag, chatId) {
 
     const chatName = preview.querySelector('#chat-name');
     chatName.innerText = name;
-
-    const msgHeader = preview.querySelector('.message-header');
-    msgHeader.querySelector('#username').innerText = 'name go here';
             
     const msgBody = preview.querySelector('.message-body');
-    msgBody.innerText = messageObj.content;
+    msgBody.innerText = messageContent;
     preview.setAttribute("id", chatId)
     changeChatOnClick(preview, tag, chatId);
+    addUsernameToMessage(uid, chatId);
 
     return preview;
 }
 
+// i can't access two paths at the same time so i need two separate functions :/
+function addUsernameToMessage(uid, elementId) {
+    const userRef = firebase.database().ref('/users/'+uid);
+    userRef.once('value', snap => {
+        const element = document.getElementById(elementId);
+        console.log(snap.key)
```

Comment:
Semicolon for consistency?  Actually, does this line (and the next) need to make it to master at all?

Reply:
fixed!